docs/source/cpp/compute.rst

```diff
@@ -153,7 +153,7 @@ Notes:
 
 * \(1) Output is a ``{"min": input type, "max": input type}`` Struct
 
-* \(2) Output is a ``{"mode": input type, "count": Int64}`` Struct
+* \(2) Output is an array of ``{"mode": input type, "count": Int64}`` Struct
```

Comment:
Explain a bit more what the array represents?

Reply:
Done